### PR TITLE
chore: cleanup repo, readme, add cli script

### LINKED ISSUE #60: chore: top of waterfall 🌊
### Background
Many PR comments for the new dockerised version are better addressed at the end of the PR waterfall. This issue tracks those.

### Details
All PR action items best suited for top of water fall are integrated.

### Acceptance criteria

- [x] refactor: modes to commands with 1:1 mapping
- [x] refactor: [properly export types](https://github.com/cowprotocol/tenderly-watch-tower/pull/39#discussion_r1336078118)
- [x] refactor: [separate chain watcher](https://github.com/cowprotocol/tenderly-watch-tower/pull/47#discussion_r1336144485)
- [x] chore: [script to run cli](https://github.com/cowprotocol/tenderly-watch-tower/pull/39#discussion_r1336068755)
- [x] #67
- [x] chore: [parameterize db location](https://github.com/cowprotocol/tenderly-watch-tower/pull/52#discussion_r1334347541)
- [x] docs: jsdoc exported interfaces, eg [here](https://github.com/cowprotocol/tenderly-watch-tower/pull/39#discussion_r1336073076)
- [x] docs: update readme
- [x] docs: docker usage

### PR DESCRIPTION
**Pull request by mfw78**
# Description
This PR finishes off feedback from #60.

# Changes

- [x] Comprehensive editorial review and adjustments to `README.md` to reflect current state of the repo 
- [x] Long-awaited addition of `cli` script into `package.json`
- [x] Parameterized the database location (configured via optional `CLI` option)
- [x] Remove all references to Tenderly

## How to test

1. Use latest `pr` `docker` image.
2. Follow instructions in `README.md` for running from a `docker` image.
3. Confirm that the respective chain achieves sync.

## Related Issues

Fixes #60

### CODE REVIEW

**Review by anxolin — APPROVED** [review]
❤️ docker command worked like a charm. Great documentation

File: src/utils/logging.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import prefix from "loglevel-plugin-prefix";
 import chalk, { Chalk } from "chalk";
 
 const DEFAULT_LOG_LEVEL = "INFO";
-const LEVELS = ["TRACE", "DEBUG", "INFO", "WARN", "ERROR", "SILENT"];
+const LEVELS = ["TRACE", "DEBUG", "INFO", "WARN", "ERROR"];
```
[the remark this answers is not in the document]
Comment (mfw78) [reply src/utils/logging.ts:11]: I must have messed up, as I thought I searched the lib and didn't see it (turns out I likely just searched the `watch-tower` repo and didn't see it). Will add back in.

File: README.md [hunk 2]
```diff
@@ -1,78 +1,51 @@
-# Watch Tower for Composable CoWs 🐮🎶
+# Watch-Tower for Composable CoWs 🐮🎶
 
-A watch tower has been implementing using standalone ethers.js. By means of _emitted Event_ and new block monitoring, conditional orders can run autonomously.
+## Overview
 
-Notably, with the `ConditionalOrderCreated` and `MerkleRootSet` events, multiple conditional orders can be created for one safe - in doing so, the actions maintain a registry of:
+`ComposableCoW` requires a watch-tower to monitor the blockchain for new orders, and to post them to the [CoW Protocol `OrderBook` API](https://api.cow.fi/docs/#/). The watch-tower is a standalone application that can be run locally as a script for development, or deployed as a docker container to a server, or dappnode.
```
[the remark this answers is not in the document]
Comment (mfw78) [reply README.md:5]: Adjusted slightly, but think that most of this is covered.

File: README.md [hunk 3]
```diff
@@ -129,8 +102,109 @@ The server exposes automatically:
...
+- `pr-<PR_NUMBER>` - the latest version of the watch-tower on the PR.
+
+As an example, to run the latest version of the watch-tower via `docker`:
+
+```bash
+docker run --rm -it \
+  ghcr.io/cowprotocol/watch-tower:latest \
+  run \
+  --rpc <rpc-url> \
+  --deployment-block <deployment-block> \
```
[the remark this answers is not in the document]
Comment (mfw78) [reply README.md:134]: I'm generally very loathe to put blocks etc into the comment as this then becomes some floating artifact of documentation as parts are updated. 

Instead, I'll mention a link to the deployments in the `ComposableCoW` repository in the Quick Start section (new section).

File: README.md [hunk 4]
```diff
@@ -1,78 +1,51 @@
...
 
-```bash
-yarn
-yarn ts-node ./src/index.ts run --rpc http://127.0.0.1:8545 --deployment-block <deployment-block> --contract-address <contract-address> --page-size 0
-```
+When a new event is discovered, the watch-tower will:
 
-### Watch tower state
+1. Fetch the conditional order(s) from the blockchain.
+2. Post the **discrete** order(s) to the [CoW Protocol `OrderBook` API](https://api.cow.fi/docs/#/).
```
[the remark this answers is not in the document]
Comment (mfw78) [reply README.md:73]: Done

**Pull request merged**